### PR TITLE
Update about.rst

### PR DESCRIPTION
**Pull request by v-h-gn**
_Use the following template. Feel free to remove any unused sections._

### About ProjectKorra
* Changes description to be PK specific instead of PK Core specific.
> * Links to any applicable GitHub Issues or Trello Cards

### Contributing
* Changes description to direct users to information below it.
* Changes reference display text to ``Contributing to the Wiki`` for clarification.
> * Links to any applicable GitHub Issues or Trello Cards

### Useful Links
* Changes text to provide more information.
> * Links to any applicable GitHub Issues or Trello Cards

### CODE REVIEW

File: source/general/about.rst [hunk 1]
```diff
@@ -2,28 +2,23 @@
...
 
-ProjectKorra was originally a Bukkit plugin that was updated for Spigot by MistPhizzle, a helper on the
-Bukkit version. Since then ProjectKorra has grown into a team that works on various projects all based
-around adapting the *Avatar: The Last Airbender* and *The Legend of Korra* universe into Minecraft.
+ProjectKorra is an organization, established by MistPhizzle in June 2014, to continue maintaining a version of Orion’s original Minecraft: The Last Airbender plugin for Bukkit. ProjectKorra’s purpose is to bring the Avatar: The Last Airbender and Avatar: The Legend of Korra universes into Minecraft, giving player an opportunity to experience the show in a new light.
 
 Contributing
 ============
-:ref:`Joining the team <jointheteam>` is the best way to contribute, but one can also contribute even when not part of the team!
+Here you can find information on how one can contribute to ProjectKorra. Although :ref:`joining the team <pkteam>` is the best way to contribute, below you can find information how to contribute as a community member!
```
Comment (ChristopherWMM) [inline source/general/about.rst]: Change `how one can` to `how you can`.

File: source/general/about.rst [hunk 2]
```diff
@@ -2,28 +2,23 @@
...
 Useful Links
 ============
-- Check out our `Website`_ for downloads, official announcements, and all the goodies!
-- Join our `Discord`_ for the best help and quickest response!
-- Follow us on `Twitter`_ to stay up to date on what is happening!
-- `GitHub`_ has all our code and is the place to contribute!
+- Visit our `Website`_, where you can find our forums, downloads, and the most recent community bulletin.
+- Join our `Discord`_ server! There you can get help from developers, post suggestions, and chat with community members!
+- Follow our `Twitter`_ for all the latest news!
+- Go to our `GitHub`_ to see all the latest developments and updates!
```
Comment (ChristopherWMM) [inline source/general/about.rst:19]: ProjectKorra is open source, perhaps mention that here?

File: source/general/about.rst [hunk 3]
```diff
@@ -2,28 +2,23 @@
 About ProjectKorra
 ==================
 
-ProjectKorra was originally a Bukkit plugin that was updated for Spigot by MistPhizzle, a helper on the
-Bukkit version. Since then ProjectKorra has grown into a team that works on various projects all based
-around adapting the *Avatar: The Last Airbender* and *The Legend of Korra* universe into Minecraft.
+ProjectKorra is an organization, established by MistPhizzle in June 2014, to continue maintaining a version of Orion’s original Minecraft: The Last Airbender plugin for Bukkit. ProjectKorra’s purpose is to bring the Avatar: The Last Airbender and Avatar: The Legend of Korra universes into Minecraft, giving player an opportunity to experience the show in a new light.
```
Comment (ChristopherWMM) [inline source/general/about.rst]: Change `universes` to just `universe`.

Comment (ChristopherWMM) [inline source/general/about.rst]: Maybe change `Orion` to `Orion304`?

Comment (vtovmasian) [reply source/general/about.rst]: oop

Comment (vtovmasian) [reply source/general/about.rst]: oh its not just orion 😅

File: source/general/about.rst (hunk 2 above)
Comment (vtovmasian) [reply source/general/about.rst:19]: I mentioned it in the description

**Review by ChristopherWMM — APPROVED** [review]
Looks good! :+1:

**Review by jphuse — APPROVED** [review]

**Pull request merged**